thirdparty/restito/src/main/java/com/xebialabs/restito/server/secure/SecureStubServer.java

```diff
@@ -51,9 +51,6 @@
 @SuppressFBWarnings("MF_CLASS_MASKS_FIELD")
 public class SecureStubServer extends StubServer {
 
-  @SuppressWarnings("WeakerAccess")
-  public static final int DEFAULT_PORT = 6666;
```

Comment:
❓ Were these two fields already declared in the parent class, so we could remove them?

Reply:
Done.